src/Library/demos/Stack/main.vala

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1 --pkg gio-2.0
+
+using Gtk;
+using Adw;
+using Gdk;
+using GLib;
+
+public void main() {
+  var root_box = workbench.builder.get_object("root_box") as Box;
```

Comment:
Use the `(Box)` instead of `as Gtk.Box` casting method in this case. Also the variables should be global and only initialized in the main function, so that they can be accessed from other functions as well.